### PR TITLE
[PRIMARY_SCREEN] [SECONDARY_SCREEN] [SWE-1658] Create typography prim…

### PR DESCRIPTION
**Pull request by whoiam460**
…ary and secondary screen #done

### CODE REVIEW

File: src/app/domains/Iconography/components/IconList/IconList.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { ICONS_LIST } from 'constants'
+import { ICONS_LIST } from 'app/__constants__'
```
Comment (Olexandr25) [inline src/app/domains/Iconography/components/IconList/IconList.js:1]: rewrite everywhere path

Comment (ArtemSuhoy) [reply src/app/domains/Iconography/components/IconList/IconList.js:1]: ~/app/

File: src/app/domains/Typography/components/TypoIntro/TypoIntro.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    <>
+      <Box mb={2}>
+        <Title variant="heading4">{title}</Title>
+      </Box>
+      <Box mb={32}>
+        <Text variant="body2" color="white-secondary">
+          {subtitle}
+        </Text>
+      </Box>
+      <Box width="384px" mb={40}>
```
Comment (ArtemSuhoy) [inline src/app/domains/Typography/components/TypoIntro/TypoIntro.js:17]: Maybe you need to move it to styled and use vars in rem

File: src/app/domains/Iconography/components/IconList/IconList.js (hunk 1 above)
Comment (whoiam460) [reply src/app/domains/Iconography/components/IconList/IconList.js:1]: i fix it in prev PR so it now on dev and when i merge this PR all will be ok

**Review by Olexandr25 — APPROVED** [review]

**Review by AnastasiaShapowalova — APPROVED** [review]

**Pull request merged**